bears-app/src/components/shared/api/mock-data/content-data.json

```diff
@@ -0,0 +1,814 @@
+{
+  "lifeEventForm": {
+    "id": "death-of-a-loved-one",
+    "timeEstimate": "60 minutes",
+    "titlePrefix": "",
+    "title": "Benefit Finder After Losing A Loved One",
+    "summary": "We are sorry about your loss. You and your family may be eligible for survivor benefits from the government following a family member's death. Learn how to find the benefits you may qualify for.",
+    "lede": "",
+    "secondaryHeading": "Steps to find benefits",
+    "icon": "",
```

Comment:
MVP would be to not have this field, use `id` to map in a switch case component with svg icons in the JS application.